test/e2e/mob_test.go

```diff
@@ -95,32 +98,110 @@ func TestMachineOSBuilder(t *testing.T) {
 	// assertion to see if the deployment object is present after setting the label
 	ctx := context.TODO()
 	err := wait.PollUntilContextTimeout(ctx, 2*time.Second, time.Minute, true, func(ctx context.Context) (bool, error) {
-		exists, err := helpers.CheckDeploymentExists(cs, "machine-os-builder", namespace)
+		exists, err := helpers.CheckDeploymentExists(cs, "machine-on-cluster-build", namespace)
```

Comment:
suggestion: Unless this was changed in `manifests/machineosbuilder/deployment.yml`, change these back to `machine-os-builder` since that's the deployment this test is looking for.